WORKSPACE

```diff
@@ -38,7 +38,7 @@ http_archive(
 # gRPC
 http_archive(
     name = "com_github_grpc_grpc",
-    urls = ["https://github.com/grpc/grpc/archive/master.tar.gz"],
+    urls = ["https://github.com/grpc/grpc/archive/master.zip"],
```

Comment:
Why change this? If the bazel bug was fixed, I think it should be its own commit.

Reply:
It seems that travis does not like the zip file.  Reverting to the tar.gz file.

Reply:
Travis (dot yml) is installing an old version of bazel which doesn't have the symlink-in-zip-file fix.

That said, I don't think it's worth updating at this time, and not without discussing what minimum version of bazel we require.